doc/minimumrequirements.md

```diff
@@ -0,0 +1,22 @@
+# Minimum requirements
```

Comment:
```suggestion
# Minimum Requirements
```

It seems we mostly use Title Case instead of sentence case in this repository for some reason so maybe let's stick to that. I asked ChatGPT and while both are correct, the most important would be consistency, I guess.

Reply:
Updated to Title Case